controllers/cloud.redhat.com/clowdenvironment_controller.go

```diff
@@ -177,12 +183,86 @@ func runProvidersForEnv(provider providers.Provider) error {
 func (r *ClowdEnvironmentReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	r.Recorder = mgr.GetEventRecorderFor("env")
 	return ctrl.NewControllerManagedBy(mgr).
-		Owns(&apps.Deployment{}).
-		Owns(&core.Service{}).
```

Comment:
why were these two lines removed?

Reply:
I thought these were the cause of the loop and removed them. It was elsewhere.

Reply:
I tried running the e2e tests locally with those two lines added back in, and there was no infinite reconciliation loop.  Is there a particular scenario I need to set up by hand to trigger the loop?

Reply:
No that should be totally resolved. It was a misunderstanding on my end. The loop was caused by the constant stream of appends we mentioned in the call this morning.